functions.php

```diff
@@ -321,7 +321,7 @@ function as_get_scheduled_actions( $args = array(), $return_format = OBJECT ) {
  *
  * @return ActionScheduler_DateTime
  */
-function as_get_datetime_object( $date_string = null, $timezone = 'UTC' ) {
+function as_get_datetime_object( $date_string = '', $timezone = 'UTC' ) {
```

Comment:
In the error logs you shared, I can see:

`[06-Jun-2022 11:20:34 UTC] PHP Deprecated:  DateTime::__construct(): Passing null to parameter #1 ($datetime) of type string is deprecated in /.../plugins/sfwd-lms/includes/lib/action-scheduler/functions.php on line 316`

Note that line number, which does not correlate with the code you are changing (line 316 is now a comment).

Is it possible you originally saw this problem when using [Action Scheduler 3.4.0](https://github.com/woocommerce/action-scheduler/blob/3.4.0/functions.php#L316) or earlier? I think the problem you encountered has actually been solved since [Action Scheduler 3.4.1](https://github.com/woocommerce/action-scheduler/blame/3.4.1/functions.php#L330) and applying this change to our current code won't actually help.

Please let me know if I'm misinterpreting, otherwise I think we can close this one.